src/tox/config/of_type.py

```diff
@@ -1,6 +1,7 @@
 """
 Group together configuration values that belong together (such as base tox configuration, tox environment configs)
 """
+import itertools as it
```

Comment:
`from itertools import product`

Reply:
Changed. How about importing from `itertools` in `src/tox/config/loader/ini/factor.py`?

Reply:
What do you mean?

Reply:
Everywhere else there's `from itertools import …`, only in that `src/tox/config/loader/ini/factor.py` there's the plain `import itertools`. I'm asking whether you'd like to have it repo-wide consistent.

Reply:
Oh must be an oversight, definitely let's keep it consistent.👌

Reply:
Okay, will add an extra commit to this PR.